stix_shifter_modules/nozomi/stix_transmission/connector.py

```diff
@@ -0,0 +1,216 @@
+from stix_shifter_utils.modules.base.stix_transmission.base_json_sync_connector import BaseJsonSyncConnector
+from stix_shifter_utils.utils.error_response import ErrorResponder
+from stix_shifter_utils.utils import logger
+from .api_client import APIClient
+import json
+import regex
+
+
+class InvalidMetadataException(Exception):
+    pass
+
+
+class Connector(BaseJsonSyncConnector):
+    NOZOMI_MAX_PAGE_SIZE = 10000
```

Comment:
Based off Nozomi's documentation, I believe this value should be lower. I do not believe this is a scenario where it must be 10,000. 

https://technicaldocs.nozominetworks.com/products/n2os/sdk/open-api-query.html

"To avoid heavy response payloads and potential performance issues, it is recommended to set a reasonable limit on the number of items per page. Generally, the number of items per page should be kept below 1000, unless there is a specific use case that necessitates a higher value."

Reply:
10000 is a maximum limit only, if the user sets the result limit to 1000, the page size will change to 1000.

Reply:
As per the recommendation followed max page size is 1000.